.travis.yml

```diff
@@ -1,13 +1,25 @@
 language: node_js
 node_js:
-  - "lts/boron"
   - "lts/carbon"
+  - "lts/dubnium"
+matrix:
+  allow_failures:
+    - node_js: "lts/dubmium"
 dist: xenial
 services:
   - mongodb
 after_success:
   - npm run test:upload-cov
   - npm run test:load
   - ".travis/build_docker.sh"
+before_install:
+  - export TZ=Africa/Johannesburg
 addon:
   srcclr: true
+notifications:
+  slack:
+    rooms:
+      - jembihealthsystems:mlQYVFbijxcZkesCt7G5VBoM
```

Comment:
I think we need to change this slack token to be one for `hearth-alerts`. This token is going to `openhim-alerts`

Reply:
Ive created a slack channel for `hearth-alerts` and created the token.
`jembihealthsystems:6VZdEFjN6eL1ZXL4Totf8wOY`